evap/staff/views.py

```diff
@@ -919,7 +919,11 @@ def semester_questionnaire_assign(request, semester_id):
 
 
 @manager_required
-def semester_preparation_reminder(request, semester_id):
+def semester_preparation_reminder(request: HttpRequest, semester_id: int) -> HttpResponse:
+    mode = request.GET.get("mode", "interactive")
+    if request.method == "POST" and mode == "text" or mode not in ["interactive", "text"]:
```

Comment:
Aren't extraneous GET parameters ignored by default? I think it would be better if we would move the `move = ` line down just before `template_data =`, it's not really important for the rest of the view